src/cpp/rtps/security/SecurityManager.cpp

```diff
@@ -1018,8 +1018,6 @@ void SecurityManager::delete_participant_stateless_message_pool()
 
         PoolConfig reader_cfg = PoolConfig::from_history_attributes(participant_stateless_message_reader_hattr_);
         participant_stateless_message_pool_->release_history(reader_cfg, true);
-
```

Comment:
Shouldn't we do a `participant_stateless_message_pool_.reset()` here?

Reply:
certainly!